Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
```

Comment:
We will (most) probably have an if statement to disable those two lines automatically if they are not required. [Something along those lines](https://github.com/ludvichzme/SFML/commit/c3f4f62f306f6f9b979b8c0cb2bfb3968712c8fa#commitcomment-9142687). ;-)